docs/reference/commandline/swarm_init.md

```diff
@@ -103,6 +105,21 @@ name, the default port 2377 will be used.
 
 This flag sets up task history retention limit.
 
+### `--max-snapshots`
+
+This flag sets the number of old Raft snapshots to retain in addition to the
+current Raft snapshots. By default, no old snapshots are retained. This option
+may be used for debugging, or to store old snapshots of the swarm state for
+disaster recovery purposes.
```

Comment:
We may want to update this part of the documentation to include this information / example / best practice; https://docs.docker.com/engine/swarm/admin_guide/#/recover-from-disaster